src/video-interface.js

```diff
@@ -218,3 +218,41 @@ export const VideoEvents = {
    */
   RELOAD: 'reloaded',
 };
+
+/**
+ * Playing States
+ *
+ * Components implementing the VideoInterface are expected to have one of these
+ * playing states.
+ *
+ * @constant {!Object<string, string>}
+ */
+export const PlayingStates = {
```

Comment:
move to `video-manager-impl` file. This enum is internal, `video-interface`  is reserved for external methods and events that video-players need to implement.